### DISCUSSION TITLE
庭灯收集与罗盘的小建议

### DISCUSSION THREAD

**Discussion by bwmgd** [TartaricAcid/TouhouLittleMaid#349] [Suggestions & Ideas]
1.刷怪塔中p点使用庭灯收集当收集满了以后就不再收集，导致p点爆出游戏卡顿，建议增加p点销毁升级或者液体形态可以导出
2.女仆经常会走丢，传送机制有时不能有效使用，建议对河童的罗盘增加指向女仆功能以方便寻找，或者单独添加传送饰品这种，回到玩家身边

**Comment by TartaricAcid** [comment]
问题 2 无法有效解决，你无法在已经卸载的区块上寻找实体
